docs/variants/novacustom_nv4x/building.md

```diff
@@ -11,46 +11,79 @@ This documents describes the procedure for compiling coreboot for NovaCustom NV4
     + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
 - Git
 
-## Procedure
+## Build Dasharo BIOS firmware
 
-The easiest way to build coreboot is to use the official Docker image.
+The easiest way to build Dasharo firmware is to use the official Docker image.
 
-Obtain the image:
+1. Obtain the image:
 
-```bash
-docker pull coreboot/coreboot-sdk:0ad5fbd48d
-```
+    ```bash
+    docker pull coreboot/coreboot-sdk:0ad5fbd48d
+    ```
 
-Clone the coreboot repository:
+1. Clone the coreboot repository:
 
-```bash
-git clone https://review.coreboot.org/coreboot.git
-```
+    ```bash
+    git clone https://github.com/Dasharo/coreboot.git
+    ```
 
-Navigate to the source code directory and checkout to the desired revision:
+    Navigate to the source code directory and checkout to the desired revision:
 
-> Replace the REVISION with one of the:
-> - `novacustom_nv4x/release` for the latest released version
-> - `novacustom_nv4x_vVERSION` (e.g. `v1.2.1`) for the given release
+    > Replace the REVISION with:
+    >
+    > - `clevo/release` for the latest released version
+    > - `novacustom_nv4x_vVERSION` (e.g. `v1.3.0`) for the given release
 
-```bash
-cd coreboot
-git remote add dasharo https://github.com/dasharo/coreboot.git
-git submodule update --init --recursive --checkout
-git fetch dasharo
-git checkout REVISION
-```
+    ```bash
+    cd coreboot
+    git checkout REVISION
+    git submodule update --init --recursive --checkout
+    ```
 
-Build the firmware:
+1. Start the coreboot-sdk Docker container:
 
-```bash
-./build.sh build
-```
+    ```bash
+    docker run --rm -it -u $UID \
```

Comment:
@mkopec could You refactor this section? Command after copying does not work at all

Reply:
Is there an error? It drops into bash inside the container on my machine

Reply:
@mkopec sorry, I have some internal error, the command works fine